includes/embeds/class-amp-gutenberg-categories-handler.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Class AMP_Gutenberg_Categories_Handler
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Gutenberg_Categories_Handler
+ *
+ * @since 1.0
+ */
+class AMP_Gutenberg_Categories_Handler extends AMP_Base_Embed_Handler {
```

Comment:
As noted above, the “Gutenberg” name is going to be dropped with the core merge so we can change it to just `AMP_Categories_Block_Handler`. But given that there is only one block that needs to be overridden, maybe it would be better to just call it `AMP_Core_Block_Handler`. This then can include the logic for the `categories` block and we can add support for any other core blocks that need special handling instead of creating a separate class for each, which I think is somewhat overkill.